cdci_data_analysis/analysis/parameters.py

```diff
@@ -448,13 +467,13 @@ def check_int_value(value, units=None,name=None):
               value=ast.literal_eval(value)
             except:
                 pass
-            value=np.int(value)
-            if type(value) == int or type(value) == np.int:
-                pass
-            elif type(value) == float or type(value) == np.float:
-                pass
-            else:
-                raise RuntimeError('type of ', name, 'not valid', type(value))
+            try:
+                # TODO a different and bigger precision (eg float64) should be considered?
+                if type(value) == float:
```

Comment:
`isinstance(value, float)` is a more conventional way, and covers cases which we might have.